docs/contributing/coding_guide.rst

```diff
@@ -952,19 +940,25 @@ in the README file of `benchmarks-repo`_.
 .. _example notebook on particles: ../notebooks/getting_started/particles.ipynb
 .. _example notebook on units: ../notebooks/getting_started/units.ipynb
 .. _extract function refactoring pattern: https://refactoring.guru/extract-method
-.. _NEP 29: https://numpy.org/neps/nep-0029-deprecation_policy.html
 .. _not a number: https://en.wikipedia.org/wiki/NaN
 .. _NumPy Enhancement Proposal 29: https://numpy.org/neps/nep-0029-deprecation_policy.html
-.. _SPEC 0: https://scientific-python.org/specs/spec-0000
+.. _Python Packaging User Guide: https://packaging.python.org
 .. _pyupgrade: https://github.com/asottile/pyupgrade
 .. _rename refactoring in PyCharm: https://www.jetbrains.com/help/pycharm/rename-refactorings.html
+.. _SPEC 0: https://scientific-python.org/specs/spec-0000
+.. _TOML: https://toml.io/en/v1.0.0
+.. _update pinned requirements: https://github.com/PlasmaPy/PlasmaPy/actions/workflows/update-pinned-reqs.yml
 .. _voila: https://voila.readthedocs.io
+.. _writing your pyproject.toml file: https://packaging.python.org/en/latest/guides/writing-pyproject-toml/
 
 .. _`astropy.units`: https://docs.astropy.org/en/stable/units/index.html
 .. |astropy.units| replace:: `astropy.units`
 
+.. _`ci_requirements/`: https://github.com/PlasmaPy/PlasmaPy/blob/main/ci_requirements
+.. |ci_requirements/| replace:: :file:`ci_requirements/`
+
+.. _`ci_requirements/README.md`: https://github.com/PlasmaPy/PlasmaPy/blob/main/ci_requirements/README.md
+.. |ci_requirements/README.md| replace:: :file:`ci_requirements/README.md`
```

Comment:
📝 This is a reStructuredText workaround that lets us have links from literal formatted code. It's...probably best not to think about it.  🤦🏻